ddtrace/_monkey.py

```diff
@@ -110,11 +110,13 @@
 _MODULES_FOR_CONTRIB = {
     "elasticsearch": (
         "elasticsearch",
+        "elasticsearch1",
         "elasticsearch2",
         "elasticsearch5",
         "elasticsearch6",
         "elasticsearch7",
         "opensearchpy",
+        "elastic_transport",
```

Comment:
Why isn't elasticsearch8 included here? An explanatory comment could help with the reader's understanding.

Reply:
Comment added

Starting with version 8 they moved the transport to a separate package, so we patch only that and nothing actually in the elasticsearch module itself.